frontend/src/scenes/session-recordings/player/sessionRecordingDataCoordinatorLogic.ts

```diff
@@ -170,15 +174,30 @@ export const sessionRecordingDataCoordinatorLogic = kea<sessionRecordingDataCoor
     })),
     selectors(({ cache }) => ({
         snapshots: [
-            (s, p) => [s.snapshotSources, s.viewportForTimestamp, p.sessionRecordingId, s.snapshotsBySources],
-            (sources, viewportForTimestamp, sessionRecordingId, snapshotsBySources): RecordingSnapshot[] => {
+            (s, p) => [
+                s.snapshotSources,
+                s.viewportForTimestamp,
+                p.sessionRecordingId,
+                s.snapshotsBySources,
+                s.featureFlags,
+            ],
+            (
+                sources,
+                viewportForTimestamp,
+                sessionRecordingId,
+                snapshotsBySources,
+                featureFlags
+            ): RecordingSnapshot[] => {
                 cache.processingCache = cache.processingCache || { snapshots: {} }
+                const discardRawSnapshots = !!featureFlags[FEATURE_FLAGS.REPLAY_DISCARD_RAW_SNAPSHOTS]
                 const snapshots = processAllSnapshots(
                     sources,
                     snapshotsBySources,
                     cache.processingCache,
                     viewportForTimestamp,
-                    sessionRecordingId
+                    sessionRecordingId,
+                    false,
```

Comment:
interesting that this enableYielding was always implicitly false before  
i thought it would be controlled by a flag :thinking_face: